### ISSUE TITLE
SSL SYSCALL error: EOF detected - PostgresSQL

### ISSUE THREAD

**Issue by moise-g** [TablePlus/TablePlus#2674]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
- PostgreSQL 13.5 on GCP Cloud SQL

3. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):

- Build 414
- macOS 12.2

4. The steps to reproduce this issue:

- Try to connect to postgres using an ssl tunnel  configured in a  config file
- This stopped working recently without any change in configuration on my part

**Comment by huyphams** [comment]
Hi @moise-g we did not change any driver code. Please try the solution here https://stackoverflow.com/questions/24130305/postgres-ssl-syscall-error-eof-detected-with-python-and-psycopg

**Comment by moise-g** [comment]
The underlying issue was caused by the gcloud sdk (which I use to create an iap tunnel my vpc). I've fixed it by reverting the gcloud sdk from 381.0.0 to 367.0.0.

Thanks for the help.

**Comment by anentropic** [comment]
I have the same problem ... version 368.0.0 of gcloud is the last one which works

I am making an "Over SSH" connection to postgres db via gcloud iap-tunnel instance

I have enabled ssh debug logging in TablePlus and attempted the connection using gcloud versions 368.0.0 and 403.0.0

Curiously the ssh debug logging output is identical in both cases. The failure seems to occur after the last ssh logging line is output, the last few lines look like:
```
ssh_socket_unbuffered_write:  Enabling POLLOUT for socket
packet_send2:  packet: wrote [type=50, len=912, padding_size=18, comp=893, payload=893]
ssh_socket_pollcallback:  Poll callback on socket 23 (POLLOUT ), out buffer 0
ssh_socket_pollcallback:  sending control flow event
ssh_packet_socket_controlflow_callback:  sending channel_write_wontblock callback
ssh_socket_pollcallback:  Poll callback on socket 23 (POLLIN ), out buffer 0
ssh_packet_socket_callback:  packet: read type 52 [len=16,padding=14,comp=1,payload=1]
ssh_packet_process:  Dispatching handler for packet type 52
ssh_packet_userauth_success:  Authentication successful
ssh_packet_userauth_success:  Received SSH_USERAUTH_SUCCESS
ssh_packet_need_rekey:  packet: [data_rekey_needed=0, out_blocks=86, in_blocks=35
```

So it seems like ssh connection is established successfully, but when using the latest version (403.0.0) of gcloud cli then TablePlus fails with:
![Screenshot 2022-09-22 at 18 15 36](https://user-images.githubusercontent.com/147840/191811731-a6fb3f4b-0276-4acb-b7c9-541f901061b4.png)
